src/util/interpolate.ts

```diff
@@ -1,6 +1,25 @@
+import {hcl, lab, RGBColor} from './color_spaces';
 import Color from './color';
 import Padding from './padding';
 
+export type InterpolationColorSpace = keyof typeof colorInterpolation;
```

Comment:
isn't this simply `'rbg' | 'hcl' | 'lab'`? if so, it's much easier to understand using these simple 3 options instead of `keyof typeof ...`.

Reply:
In this case, it is important that typescript understands it and that it is in low maintenance mode :D